frontend/enums/index.ts

```diff
@@ -39,6 +39,7 @@ export enum Paths {
   Invitation = '/invitation',
   ForInvestors = '/for-investors',
   ForProjectDevelopers = '/for-project-developers',
+  ComingSoon = '/coming-soon',
```

Comment:
I really don't think we should have a dedicated (and in the future indexed) `/coming-soon` page/url. 
See my comments [here](https://github.com/Vizzuality/heco-invest/pull/429#pullrequestreview-1053985857).

Reply:
I did it wrong . I will fix it